src/Plugin/GitLab/Controller.php

```diff
@@ -65,6 +66,7 @@ public function updateAction(Application $app, Request $request, $id)
             return new Response();
         }
 
+        $config->setFqnPrefix($request->get('gitlab_token'));
         $config->setToken($request->get('gitlab_token'));
```

Comment:
I think you meant to have `$config->setFqnPrefix($request->get('fqn_prefix'));` here.